examples/getting_started/2_Pipeline.ipynb

```diff
@@ -500,7 +651,31 @@
+    "Because Datashader is designed to reveal the properties of your dataset without making any assumptions about its range or form, the default transformation is `eq_hist`; you can select any other `how` option for `tf.shade` if needed and appropriate for your particular dataset.\n",
+    "\n",
+    "Note that histogram equalization dynamically scales the input aggregate values into the _entire_ colormap range, which is appropriate when you have a large dataset, using all available colors to convey properties of the data being displayed. However, in small-data cases where you are looking at widely separated, rarely overlapping points, most datapoints would end up being colored with the lowest colormap value (here a barely visible light blue color), with the highest (and thus most visible) colormap value reserved for the occasional overlapping points:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "zoom_agg = ds.Canvas(plot_width=300, plot_height=300, x_range=(1.92,1.95), y_range=(1.92,1.95)).points(df, 'x', 'y')\n",
+    "zoom = tf.spread(zoom_agg, px=3)\n",
+    "\n",
+    "tf.Images(tf.shade(zoom, how='eq_hist', name=\"eq_hist rescale_discrete_levels=False\", rescale_discrete_levels=False),\n",
+    "          tf.shade(zoom, how='eq_hist', name=\"eq_hist rescale_discrete_levels=True\",  rescale_discrete_levels=True))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To support interactive usage where you want the full colormap range available when plotting many points, but for individual points to be more visible when there are only a few discrete levels (two, in the plots above), as of Datashader 0.14 you can specify `rescale_discrete_levels=True` to use only the upper end of the colormap range, letting values be distinguished while making all points similarly visible (as in the `eq_hist rescale_discrete_levels=True` plot above).\n",
+    "\n",
+    "Of course, the `how` options like logarithmic scaling and histogram equalization can be considered part of the ***Transformation*** stage, but in practice these well-defined transformations are applied during colormapping (\"shading\") so that the raw aggregate values can be kept in their untransformed state suitable for direct numerical lookup during the subsequent ***Embedding*** stage, so that e.g. interactive hovering can report the untransformed values. See the [shade docs](https://datashader.org/api.html#datashader.transfer_functions.shade) for more details on the `how` functions. \n",
```

Comment:
```suggestion
    "Of course, the `how` options like logarithmic scaling and histogram equalization can be considered part of the ***Transformation*** stage, but in practice these well-defined transformations are applied during colormapping (\"shading\") so that the raw aggregate values can be kept in their untransformed state suitable for direct numerical lookup during the subsequent ***Embedding*** stage. That way, interactive hovering and colorbars can report the accurate untransformed values. See the [shade docs](https://datashader.org/api.html#datashader.transfer_functions.shade) for more details on the `how` functions. \n",
```